src/Table/index.jsx

```diff
@@ -73,14 +170,22 @@ Table.propTypes = {
       PropTypes.string,
       PropTypes.element,
     ]).isRequired,
+    columnSortable: isRequiredIf(PropTypes.bool, props => props.tableSortable),
```

Comment:
I'm not sure that this conditional require and the one below it actually work correctly.